misc/formatter/modules/formatter-cli/src/main/resources/META-INF/services/org.ballerinalang.tool.BLauncherCmd

```diff
@@ -0,0 +1 @@
+
```

Comment:
Should there be an entry in this ?

Reply:
This was done to keep the previous ballerina-formatter CLI working instead of this one which is for the new parser. But now the previous ballerina-formatter CLI tool is removed and this one is enabled.